accelerator/core/apis.h

```diff
@@ -169,15 +169,17 @@ status_t api_send_mam_message(const ta_config_t* const info, const iota_config_t
  * fields include address, value, tag, and message. This API would also try to
  * find the transactions after bundle sent.
  *
- * @param core[in] Pointer to Tangle-accelerator core configuration structure
+ * @param[in] core Pointer to Tangle-accelerator core configuration structure
+ * @param[in] iota_service IRI node end point service
```

Comment:
I think the `end point` should not be used since we have another `endpoint`.
And since we might use `Hornet`, the `IRI node` description might not be accurate?

Reply:
How about rewriting it to `IOTA node service`?

Reply:
Sounds good.

Reply:
I found out the original description is `iota connection structure` in `ta_core_s`.
Please use the suitable one.